chains/rootstocktestnet/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://rootstock-testnet.blockscout.com/api
+    family: blockscout
+    name: Blockscout
+    url: https://rootstock-testnet.blockscout.com
+blocks:
+  confirmations: 1
+  estimateBlockTime: 30
+  reorgPeriod: 4
+chainId: 31
+deployer:
+  name: Rootstock
+  url: https://rootstock.io
+displayName: Rootstock Testnet
+domainId: 31
+isTestnet: true
+name: rootstocktestnet
+nativeToken:
+  decimals: 18
+  name: tRBTC
+  symbol: tRBTC
+protocol: ethereum
+rpcUrls:
+  - http: https://rpc.testnet.rootstock.io/YZIZHxGCQAryubOWH7LjdtjNkSDb5J-T
```

Comment:
This looks like an API key. Are you sure you meant to include it here in this public repo?

Reply:
@jmrossy Yes. We are sure to include it in public repo. We discussed and well thought to use api key here. 
Adding @alepc253 to this discussion.

Reply:
@alepc253 👆

Reply:
A comment would be a good idea! Thankfully this is YAML, not JSON, so you can add any comments you like above this line

Reply:
Added comments. Thanks for this suggestion. 👍